src/seedu/addressbook/commands/StorageSizeCommand.java

```diff
@@ -0,0 +1,17 @@
+package seedu.addressbook.commands;
+
+public class StorageSizeCommand extends Command {
+    public static final String COMMAND_WORD = "storagesize";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Checks the size of storage file.\n"
+            + "Example: " + COMMAND_WORD;
+
+    private static final String MESSAGE_DISPLAY_FILE_SIZE = "Your storage file size is: %1$s bytes.";
+
+    public StorageSizeCommand() {}
```

Comment:
Please read the [Coding Standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#layout), what should the layout of this constructor be? :)